src/test_l1_dsHost.c

```diff
@@ -612,8 +645,12 @@ void test_l1_dsHost_negative_dsGetHostEDID(void) {
 
     // Step 06: dsGetHostEDID() Call after termination
     result = dsGetHostEDID(edid, &length);
+#ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.